astropy/coordinates/baseframe.py

```diff
@@ -448,6 +448,7 @@ class BaseCoordinateFrame(object):
     # Default empty frame_attributes dict
 
     def __init__(self, *args, **kwargs):
+        copy = kwargs.pop('copy', True)
```

Comment:
Maybe add just a line or two in the docstring that indicates that the copy kwarg is processed by `BaseCoordinateFrame`?  If not it's conceivable that someone writing their own frame might not know about it.